ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/config/AuditYamlConfigurationLoader.java

```diff
@@ -13,59 +13,71 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.ericsson.bss.cassandra.ecaudit.filter.yaml;
+package com.ericsson.bss.cassandra.ecaudit.config;
 
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URL;
-import java.nio.file.Files;
 import java.util.Properties;
 
-import org.apache.cassandra.exceptions.ConfigurationException;
-import org.apache.cassandra.io.util.FileUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+
+import org.apache.cassandra.exceptions.ConfigurationException;
+import org.apache.cassandra.io.util.FileUtils;
 import org.yaml.snakeyaml.Yaml;
 import org.yaml.snakeyaml.constructor.Constructor;
 import org.yaml.snakeyaml.error.YAMLException;
 
 /**
- * Implements an {@link AuditConfigurationLoader} that loads configuration from a YAML file.
+ * Loads configuration from a YAML file.
  */
-public final class AuditYamlConfigurationLoader implements AuditConfigurationLoader
+public final class AuditYamlConfigurationLoader
 {
     private static final Logger LOG = LoggerFactory.getLogger(AuditYamlConfigurationLoader.class);
 
-    private final static String DEFAULT_CONFIG_FILE = "/etc/cassandra/conf/audit.yaml";
+    private static final String DEFAULT_CONFIG_FILE = "audit.yaml";
 
     /**
-     * Property for the path to a {@link AuditConfig} file that configures audit logging.
+     * Property for the path to a {@link AuditYamlConfig} file that configures audit logging.
      */
-    public final static String PROPERTY_CONFIG_FILE = "com.ericsson.bss.cassandra.eaudit.config";
+    public static final String PROPERTY_CONFIG_FILE = "com.ericsson.bss.cassandra.ecaudit.config";
 
     private final Properties properties;
 
     /**
      * Construct a new {@link AuditYamlConfigurationLoader} instance that loads configuration from the provided
      * properties
-     * @param properties
-     *            where to get the property file url from
+     *
+     * @param properties where to get the property file url from
      */
     private AuditYamlConfigurationLoader(Properties properties)
     {
         this.properties = new Properties(properties);
     }
 
-    @Override
-    public AuditConfig loadConfig() throws ConfigurationException
+    /**
+     * Loads a {@link AuditYamlConfig} object.
```

Comment:
Just a comment regarding the use of "@link" in javadoc. I usually try to avoid it as much as possible.
All types (classes) that is part of the method signature (including return type) will always have links generated to them automagically. So There is no need to have extra links written "manually", there is always the risk that these links will be forgotten (to be updated) if the method signature changes... And then these links will be faulty. 
Also is makes the javaDoc look more "messy" when you look at it in the IDE.

So I only use @link when it is to something that is not part of the method signature.

Reply:
Good advice! Totally agree that link look messy in the IDE. Fixed!